src/components/Login/Login.js

```diff
@@ -1,13 +1,14 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import classNames from 'classnames';
 import { connect } from 'react-redux';
 import { reduxForm, Field, Form, formValueSelector } from 'redux-form';
+import _ from 'lodash';
```

Comment:
Instead of importing _all_ of lodash. Let's just import what we need.
```suggestion
import isEmpty from 'lodash/isEmpty';
```